OKR/PROJECT.md

```diff
@@ -2,8 +2,30 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
-## 2018 Q4
+## 2019 Q1
 
+- **The IPFS Org gets a fully dedicated Package Managers Working Group**
+  - `P0` - @daviddias - A Package Manager Working Group Roadmap is created
+  - `P1` - @??? - Onboard 2+ contributors to this Working Group       
+- **Support and guide the IPFSConf planning and Content Curation**
+  - `P0` - @daviddias - The IPFSConf is scheduled, announced and there is a CFP running. The IPFSConf Org Team is supported by the IPFS org.
+  - `P0` - @momack2 - The IPFS Org (Working Groups & Core Contributors) has a unified plan with goals for IPFSConf
+- **The IPFS Working Groups feel supported and they have the information they need to make the best decisions**
+  - `P0` - @mikeal - We gather, process and present metrics for the IPFS Project  
+  - `P1` - @parkan - Ship the first IPFS User Quarterly Report. It documents achievements and demos of new capabilities
+  - `P1` - @momack2 - Working group roadmaps are solidified, presented, and used to chart Q1 and Q2 planning
+- **The IPFS Community is supportive, responsive, and easy to get onboarded to**
+  - `P0` - @parkan - There are zero dropped/mishandled collabs  
+  - `P1` - @parkan - Biweekly updates published to IPFS working groups highlighting patterns in top developer needs
+  - `P1` - @daviddias - The Working Groups are listed on the IPFS Website (ref: ipfs/project#14)
+  - `P1` - @mikeal - Improve the onboarding experience successfully, which translates to seeing 3+ new contributors engaged per Working Group until the end of the Quarter 
```

Comment:
This isn't a practical goal for the quarter. We can't tackle the onboarding experience of every WG at the same time, and we shouldn't expect to see a rise in new external contributors engaged in the same quarter we're actively making the improvements.

The Community WG is making doc and onboarding improvements in `js-ipfs` this quarter, and taking on some broader documentation improvements, in this quarter.

Reply:
Understood. Removed it as this work is already being tracked on the current Community WG then.